src/Version.php

```diff
@@ -20,12 +20,14 @@ class Version
     /** @var bool */
     private $versionIsTagged;
 
+    const NO_REFERENCE_TEXT = '{no reference}';
```

Comment:
We now require 7.1, so we can make the visibility explicit: 
```suggestion
    public const NO_REFERENCE_TEXT = '{no reference}';
```